src/Skeletor/Packages/BehatBehatPackage.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Skeletor\Packages;
+
+use Skeletor\Frameworks\Framework;
+
+class BehatBehatPackage extends Package
+{
+    public function setup()
+    {
+        $this->setInstallSlug("behat/behat");
+        $this->setName("Behat Behat");
+    }
+
+    public function configure(Framework $activeFramework)
```

Comment:
True, what is the best practice. Because we define `configure` on our packageinterface it must be defined in the child. But like you said, its probably not always going to be used.

Should i remove it from the interface?